_data/changelogs/site-alert.yml

```diff
@@ -19,4 +19,33 @@ changelogURL:
   #   githubPr:
   #   githubRepo:
   #   versionUswds:
-# end changelog item
\ No newline at end of file
+
+items:
+  # 3.1.0 release
+  - date: 2022-10-19
+    summary: Updated the alignment of site alert content to visually match alignment with banner content.
+    summaryAdditional: The alert component was also updated to match banner content alignment.
+    isBreaking: false
+    affectsAccessibility: false
+    affectsAssets: false
+    affectsGuidance: false
+    affectsJavascript: false
+    affectsMarkup: false
+    affectsStyles: true
+    githubPr: 4922
+    githubRepo: uswds
+    versionUswds: 3.1.0
```

Comment:
This should be 3.2.0

Reply:
Fixed in 04eb195e22188a8d578f6ddf087b9cca7bb7ddb1.